app/views/Project/Tagging/Sources/BulkUpload/index.tsx

```diff
@@ -77,7 +77,13 @@ function BulkUpload(props: Props) {
     // NOTE: If a lead is removed or saved, uploaded files are not being removed at the moment
     const [uploadedFiles, setUploadedFiles] = useState<FileUploadResponse[]>([]);
     const [selectedLead, setSelectedLead] = useState<string | undefined>();
-    const [leadClientIds, setLeadClientIds] = useState<string[] | undefined>();
+    /*
+     * NOTE need to use ref instead of state
+     *  https://github.com/apollographql/apollo-client/pull/9226
+     * the above provides the solution to the issue.
+     * Once it is released we can restore to using previous code.
+     */
```

Comment:
We should have originally used ref instead of state, so we can skip adding this comment